docs/data-ai/capture-data/lorawan.md

```diff
@@ -461,19 +503,138 @@ await node.doCommand({'calibrate_orp': {}});
 ```typescript
 const node = new SensorClient(client, "<your_node_name>");
 
-// calibrate pH probe
-await node.doCommand({ calibrate_ph: {} });
-// calibrate electrical conductivity probe
-await node.doCommand({ calibrate_ec: {} });
-// calibrate turbidity probe
-await node.doCommand({ calibrate_t: {} });
-// calibrate ORP (Oxidation-Reduction Potential) probe
-await node.doCommand({ calibrate_orp: {} });
+await node.doCommand({ <command>: <value> });
 ```
 
 {{% /tab %}}
 {{< /tabs >}}
 
+{{% /tab %}}
+{{< /tabs >}}
+
+The Dragino WQS-LB supports the following calibration commands:
+
+- `calibrate_ph`
+- `calibrate_ec`
+- `calibrate_t`
+- `calibrate_orp`
+
+Use the calibration processes below to calibrate your sensor with these commands:
+
+#### Calibrate the pH probe
```

Comment:
Is this a common enough thing that we want to cover it here? Leaving that up to you but it feels odd to me

Reply:
It is a requirement to properly use this sensor model, which is one of our small number of officially supported sensors. And the pattern is generalizable across all other sensors that require calibration. So I'm leaning 'include' unless someone feels really strongly otherwise. I understand how it might feel odd but I think it's important to draw attention to this so folks don't wind up with a lot of uncalibrated sensors with unreliable readings.